SDL/Hit.cuh

```diff
@@ -1,15 +1,10 @@
 #ifndef Hit_cuh
 #define Hit_cuh
 
+#include <alpaka/alpaka.hpp>
 #ifdef __CUDACC__
-#define CUDA_HOSTDEV  __host__ __device__
-#define CUDA_DEV __device__
-#define CUDA_CONST_VAR __device__
 #define CUDA_G __global__
```

Comment:
Why does this stay here while it is removed from the other files?

Reply:
CUDA_G is defined for __global__ functions (i.e. kernels), this just focuses on moving external functions (__device__, __host__, or __device__ __host__) over.

Reply:
I see. I guess part of my confusion then comes from the fact that the same line is removed from other files, see [this comment](https://github.com/SegmentLinking/TrackLooper/pull/217#discussion_r1008675383) for example.